npc/re/merchants/barters/enchan_illusion_16_2.yml

```diff
@@ -0,0 +1,231 @@
+            Item: Deep_Sea_Starfish
+            Amount: 150
+          - Index: 2
+            Item: IllusionStone
+            Amount: 120
+      - Index: 2
+        Item: Brionac_IL
+        RequiredItems:
+          - Index: 0
+            Item: Brionac
+            Refine: 9
+          - Index: 1
+            Item: Deepsea_Fish_Tail
+            Amount: 150
+          - Index: 2
+            Item: IllusionStone
+            Amount: 120
+      - Index: 3
+        Item: KatarOfCold_Icicle_IL
+        RequiredItems:
+          - Index: 0
+            Item: Katar_Of_Cold_Icicle_
+            Refine: 9
+          - Index: 1
+            Item: Deep_Sea_Crab
+            Amount: 150
+          - Index: 2
+            Item: IllusionStone
+            Amount: 120
+      - Index: 4
+        Item: Death_Guidance_IL
+        RequiredItems:
+          - Index: 0
+            Item: Death_Guidance
+            Refine: 9
+          - Index: 1
+            Item: Rotten_Deepseafish
+            Amount: 150
+          - Index: 2
+            Item: IllusionStone
+            Amount: 120
+      - Index: 5
+        Item: Zephyrus_IL
+        RequiredItems:
+          - Index: 0
+            Item: Zephyrus_
+            Refine: 9
+          - Index: 1
+            Item: Abysmal_Clam
+            Amount: 150
+          - Index: 2
+            Item: IllusionStone
+            Amount: 120
+      - Index: 6
+        Item: Saint_Robe_IL
+        RequiredItems:
+          - Index: 0
+            Item: Saint_Robe_
+            Refine: 9
+          - Index: 1
+            Item: Deepsea_Fish_Tail
+            Amount: 100
+          - Index: 2
+            Item: E_Abyss1
+            Amount: 20
+          - Index: 3
+            Item: IllusionStone
+            Amount: 90
+      - Index: 7
+        Item: Water_Sprits_Armor_IL
+        RequiredItems:
+          - Index: 0
+            Item: Water_Sprits_Armor_
+            Refine: 9
+          - Index: 1
+            Item: Deep_Sea_Crab
+            Amount: 100
+          - Index: 2
+            Item: E_Abyss1
+            Amount: 20
+          - Index: 3
+            Item: IllusionStone
+            Amount: 90
+      - Index: 8
+        Item: Chain_Mail_IL
+        RequiredItems:
+          - Index: 0
+            Item: Chain_Mail_
+            Refine: 9
+          - Index: 1
+            Item: Deep_Sea_Starfish
+            Amount: 100
+          - Index: 2
+            Item: E_Abyss1
+            Amount: 20
+          - Index: 3
+            Item: IllusionStone
+            Amount: 90
+      - Index: 9
+        Item: Morrigane_Helm_IL
+        RequiredItems:
+          - Index: 0
+            Item: Morrigane's_Helm
+            Refine: 9
+          - Index: 1
+            Item: Deep_Sea_Crab
+            Amount: 100
+          - Index: 2
+            Item: E_Abyss1
+            Amount: 50
+          - Index: 3
+            Item: IllusionStone
+            Amount: 100
+      - Index: 10
+        Item: Morrigane_Manyeau_IL
```

Comment:
This should be a `t`, not a `y`.

Also, you may want to double check `re/item_db_equip.yml` in main to ensure that the the Morrigane `_IL` items correspond with the barter entries, and vice-versa; I had to go back and re-tool my instance just a bit because the Aegis Names either had or didn't have apostrophes (personally, I'm of the mindset that AegisName entries shouldn't have apostrophes, but this is just me nit-picking.)

Reply:
Unfortunately, we don't come up with the Aegis names, they are created by Gravity. As you can see under the English name of the item [here](https://www.divine-pride.net/database/item/480054/illusion-morriganes-manteau-1). There are many that have typos like this and once Gravity gives the name it's pretty much set in stone.

Reply:
Ah, understandable.